1:30 - Update Events (#890)
FusionAuth/fusionauth-site: site/docs/v1/tech/events-webhooks/event-bodies/user-registration-update.adoc

:type: user.registration.update
== User Registration Update

[NOTE.since]
====
This event has been available since 1.6.0
====

This event is generated when a user registration has been updated.

[source,shell]
.Event type
[subs="attributes"]
----
{type}
----

=== Event Scope

include::../_tenant-scoped-event.adoc[]

> 

Reviewer: Add `Transaction Compatible` section.

=== Event Body

[.api]
[field]#event.applicationId# [type]#[UUID]#::
The unique Id of the Application for which the user has now been registered.

[field]#event.createInstant# [type]#[Long]#::
The link:/docs/v1/tech/reference/data-types/#instants[instant] that the event was generated.

[field]#event.id# [type]#[UUID]#::
The unique Id of the event. You may receive an event more than once based upon your transaction settings. This Id may be used to identify a duplicate event.

include::_event-info.adoc[]

[field]#event.original# [type]#[Object]#::
The original registration prior to being updated. See the link:/docs/v1/tech/apis/registrations/[Registration API] for property definitions and example JSON.

[field]#event.registration# [type]#[Object]#::

> The user registration that has been updated. See the link:/docs/v1/tech/apis/registrations/[Registration API] for property definitions and example JSON.

Reviewer: ```suggestion
The user registration with the current, updated values. See the link:/docs/v1/tech/apis/registrations/[Registration API] for property definitions and example JSON.
```
